internal/acceptance/grant_test.go

```diff
@@ -54,6 +54,13 @@ resource "databricks_external_location" "some" {
 	comment         = "Managed by TF"
 }
 
+resource "databricks_grant" "metastore" {
```

Comment:
could you comment this block out for now (we need to exclude this from running in parallel with the grants test first)

Reply:
Sure.

Reply:
Done